src/components/home/components/Home.jsx

```diff
@@ -20,6 +21,9 @@ export const Home = () => {
     return (
         <>
             <main>
+                <Helmet>
+                  <title>{ "AltDotNg | Home" }</title>
+                </Helmet>
```

Comment:
I believe it will be cleaner if you applied the Helmet to `src/components/pages/components`. All the pages are there and it will be easier handling this logic there. I also suggest you add other meta tags.